docs/guides/istio.md

```diff
@@ -1,39 +1,325 @@
-## Istio
+---
+title: Istio
+lang: en-US
+meta:
+  - name: keywords
+    content: >-
+      pomerium, identity access proxy, istio, traffic management, policy,
+      mutual authentication, authorization, kubernetes
+description: >-
+  Integrate the Pomerium Ingress controller with an Istio service mesh for full mutual authentication in your cluster.
+---
 
-[istio]: https://github.com/istio/istio
-[certmanager]: https://github.com/jetstack/cert-manager
-[grafana]: https://github.com/grafana/grafana
+# Istio with Pomerium
+
+Istio provides application-aware networking via a service mesh and control plane. When configured with the [Pomerium Ingress Controller] for kubernetes, this enables authorization (**authZ**) and authentication (**authN**) of [east-west traffic] in your internal network bringing you closer to complete [zero trust].
```

Comment:
Do we want authZ, authN, east-west traffic to link to the glossary? Just my thought.

Reply:
I'll add those links once the glossary is published.